reliability-engineering/pipelines/tasks/concourse-land-workers.yml

```diff
@@ -22,7 +24,7 @@ run:
   - |
     echo "configuring fly..."
     export PATH="$PATH:/opt/resource"
-    FLY_PASSWORD=$(cat concourse-provider-config/main_team_password)
+    [ -z "$FLY_PASSWORD" ] && FLY_PASSWORD=$(cat concourse-provider-config/main_team_password)
```

Comment:
Could we have some kind of comment that explains what happens in the doesn't-have-a-`concourse-provider-config` case? I'm not sure I understand this.